lib/ovirt/template.rb

```diff
@@ -152,9 +152,7 @@ def build_clone_xml(options)
             end
           end
 
-          xml.os(:type => options[:os_type] || os_type) do
-            xml.boot(:dev => 'hd')
-          end
+          xml.os(:type => options[:os_type] || os_type)
```

Comment:
@bdunne Will this work when cloning uses the blank_template?  [lib/ovirt/template.rb#L109](https://github.com/ManageIQ/ovirt/blob/7184f6f36401cb00d4b233c60dbbc3f90cf3feb1/lib/ovirt/template.rb#L109)